src/shared/sam/activation.ts

```diff
@@ -28,12 +34,16 @@ import { addSamDebugConfiguration } from './debugger/commands/addSamDebugConfigu
 import { SamDebugSession } from './debugger/samDebugSession'
 import { AWS_SAM_DEBUG_TYPE } from './debugger/awsSamDebugConfiguration'
 
+const STATE_NAME_SUPPRESS_YAML_PROMPT = 'dontShowYamlPromptEver'
```

Comment:
nit: all of these "dontShow..." symbols/names could be `neverShow` for consistency (and to avoid the "dont...ever" case).

Reply:
I see that 7503c34cd69085d49fea7c6514b53284a5c6e908 changed this case, but there are still many other cases...

Reply:
Fair, got lazy. I blame Monday morning.